config/sync/filter.format.uswds.yml

```diff
@@ -26,7 +26,7 @@ filters:
     status: false
     weight: -10
     settings:
-      allowed_html: '<a href hreflang> <em> <strong> <cite> <blockquote cite> <code> <ul type> <ol start type> <li> <dl> <dt> <dd> <h2 id class> <h3 id class> <h4 id class> <h5 id class> <h6 id class> <img src alt data-entity-type data-entity-uuid data-align data-caption> <table class style width height> <caption> <tbody> <thead> <tfoot> <th scope> <td> <tr scope> <u> <s> <sup> <sub> <drupal-media data-entity-type data-entity-uuid> <div id class data-*> <button class aria-expanded aria-controls>'
+      allowed_html: '<a class="usa-button usa-button--secondary usa-button--big"> <p class="usa-intro"> <h2 class="font-serif-lg text-bold text-accent-cool-darker">'
```

Comment:
I notice we were allowing quite a few tags that this removes. Probably we should keep them, unless you know a reason to do otherwise? 
I'm particularly intrigued by the fact that the 'a' and 'h2' attributes allowed here are different.